JS/Planes/experimental-plane.js

```diff
@@ -0,0 +1,12 @@
+const Plane = require('./plane');
```

Comment:
Please rename a file: experimentalPlane.js -> ExperimentalPlane.js

Reply:
it was named "experimental-plane.js" do I need to rename it in camelcase?

Reply:
It should be named as `ExperimentalPlane.js`.